src/_blog/cryptotls.md

```diff
@@ -0,0 +1,52 @@
+
+This means that crypto/tls needed to support adding data to session tickets, both the client and server sides. It turned out that the API required for this would also solve a large number of other longstanding [issues related to session tickets](https://github.com/golang/go/issues/60105).
+
+This is only one of the problems we had to solve to make 0-RTT work. Another complication arises from the fact that the server can reject 0-RTT for any reason; typically for DoS protection or because it doesn’t consider the QUIC transport parameters acceptable anymore. In that case, 0-RTT packets are discarded and the handshake continues as a regular (1-RTT) QUIC handshake. This necessitated a new [API for deciding about 0-RTT rejection](https://github.com/golang/go/issues/60107) and for clients to be informed of it.
+
+## Current Status
+
+After an intense period of collaboration and development, we are thrilled to have all these changes included in the Go 1.21 release. The implementation of these proposals in the standard library was performed by [Damien Neil](https://github.com/neild) and [Filippo Valsorda](https://github.com/FiloSottile). On the quic-go side, we made use of the new APIs as soon as the changes were published. This allowed us to use quic-go's test suite to stress-test the implementation. This was really valuable, since quic-go’s test suite is quite extensive and covers a large number of corner cases. And it indeed found a small issue, which we subsequently [fixed](https://go-review.googlesource.com/c/go/+/498215).
+
+And while we have a small workaround for one remaining [issue](https://github.com/golang/go/issues/60506), we are hopeful of a resolution in the next release.
+
+The newly introduced changes have been released in [quic-go v0.37](https://github.com/quic-go/quic-go/releases/tag/v0.37.0), and in [go-libp2p v0.30](https://github.com/libp2p/go-libp2p/releases/tag/v0.30.0). We anticipate that these changes will work seamlessly once users update their dependencies and compiler version, but please let us know if you run into any problems!
```

Comment:
Describe how to let us know?  (file a go-libp2p issue?)

Is it worth emphasizing again that a consumer can now perform go-libp2p updates independent of go updates?

Reply:
Done.